src/modules/orders/index.ts

```diff
@@ -1,3 +1,3 @@
-import { Orders } from "./Orders";
+import Orders from "./Orders";
```

Comment:
I would suggest to not export using default imports from such index files.
I would do: `export { default as Orders } from "./Orders"`.